cta_dbt_helper.sh

```diff
@@ -0,0 +1,188 @@
+#!/usr/bin/env bash
+
+####  FUNCTIONS  ####
+copy_dbt_from_airbyte() {
+
+    # Set environment vars for copy script
+    ALLOWED_ENVIRONMENTS="dev\nprod"
+
+    ENVIRONMENT=$(echo -e $ALLOWED_ENVIRONMENTS | gum filter --placeholder "Which Airbyte Environment should be used?")
+    WORKSPACE_ID=$(gum input --placeholder "Airbyte Workspace ID ex. 814")
+    
+    # Prompt if Cacher Snippet should be downloaded
+    if gum confirm "Download Cacher script to copy an Airbyte workspace?"; then
+        # Get Cacher creds from user
+        CACHER_API_KEY=$(gum input --placeholder "Enter your Cacher API Key -> https://app.cacher.io/enter?action=view_api_creds")
+        CACHER_API_TOKEN=$(gum input --placeholder "Enter your Cacher API Token -> https://app.cacher.io/enter?action=view_api_creds")
+        CACHER_SNIPPET_GUID="2b77280d537736f980f9"
```

Comment:
Let's move this to a Github Secret 👀

Reply:
Hmm, yeah you are right I kinda forgot that's how Github Secrets work, and you're also right this isn't actually a Secret-Secret - can we just add some commentary explaining what the script does so if a curious onlooker sees this they will know what we are doing?